scanner/src/funTest/kotlin/HttpCacheTest.kt

```diff
@@ -73,41 +82,165 @@ class HttpCacheTest : StringSpec() {
         }
     }
 
+    private val id = Identifier("provider", "namespace", "name", "version")
+
+    private val downloadTime1 = Instant.EPOCH + Duration.ofDays(1)
+    private val downloadTime2 = Instant.EPOCH + Duration.ofDays(2)
+    private val downloadTime3 = Instant.EPOCH + Duration.ofDays(3)
+
+    private val provenanceWithSourceArtifact = Provenance(
+            downloadTime = downloadTime1,
+            sourceArtifact = RemoteArtifact("url", "hash", HashAlgorithm.SHA1)
+    )
+    private val provenanceWithVcsInfo = Provenance(
+            downloadTime = downloadTime2,
+            vcsInfo = VcsInfo("type", "url", "revision", "path")
+    )
+    private val provenanceEmpty = Provenance(downloadTime3)
+
+    private val scannerSpecs1 = ScannerSpecification("name 1", "version 1", "config 1")
+    private val scannerSpecs2 = ScannerSpecification("name 2", "version 2", "config 2")
+
+    private val scannerStartTime1 = downloadTime1 + Duration.ofMinutes(1)
+    private val scannerEndTime1 = scannerStartTime1 + Duration.ofMinutes(1)
+    private val scannerStartTime2 = downloadTime2 + Duration.ofMinutes(1)
+    private val scannerEndTime2 = scannerStartTime2 + Duration.ofMinutes(1)
+
+    private val scanSummaryWithFiles = ScanSummary(
+            scannerStartTime1,
+            scannerEndTime1,
+            1,
+            sortedSetOf("license 1.1", "license 1.2"),
+            sortedSetOf("error 1.1", "error 1.2")
+    )
+    private val scanSummaryWithoutFiles = ScanSummary(
+            scannerStartTime2,
+            scannerEndTime2,
+            0,
+            sortedSetOf(),
+            sortedSetOf()
+    )
+
+    private val rawResultWithContent = jsonMapper.readTree("\"key 1\": \"value 1\"")
+    private val rawResultEmpty = EMPTY_NODE
+
     init {
-        "HTTP GET returns what was PUT" {
+        // TODO: Replace with tests for new  API
```

Comment:
Nit: Double-space before `API` and no `.` after it.

Reply:
Removed the comment because the task was done.